docs/OV_Runtime_UG/supported_plugins/CPU.md

```diff
@@ -195,51 +198,52 @@ See [preprocessing API guide](../preprocessing_overview.md) for more details.
     * boolean
 @endsphinxdirective
 
-### Model caching
-The CPU device plugin supports Import/Export network capability. If model caching is enabled via the common OpenVINO™ `ov::cache_dir` property, the plugin will automatically create a cached blob inside the specified directory during model compilation.
-This cached blob contains partial representation of the network, having performed common runtime optimizations and low precision transformations.
-At the next attempt to compile the model, the cached representation will be loaded to the plugin instead of the initial IR, so the aforementioned steps will be skipped.
-These operations take a significant amount of time during model compilation, so caching their results makes subsequent compilations of the model much faster, thus reducing first inference latency (FIL).
+### Models Caching
+CPU plugin supports Import/Export network capability. If the model caching is enabled via common OpenVINO™ `ov::cache_dir` property, the plugin will automatically create a cached blob inside the specified directory during model compilation.
+This cached blob contains some intermediate representation of the network that it has after common runtime optimizations and low precision transformations.
+The next time the model is compiled, the cached representation will be loaded to the plugin instead of the initial IR, so the aforementioned transformation steps will be skipped.
```

Comment:
```suggestion
The next time the model is compiled, the cached representation will be loaded to the plugin instead of the initial OpenVINO IR, so the aforementioned transformation steps will be skipped.
```